### ISSUE TITLE
Download and use of phyloFlash database in bactopia

### ISSUE THREAD

**Issue by kusandeep** [bactopia/bactopia#158]
Hi Robert,
another query/help. I would like to run phyloflash on my samples and for that I will need to download phyloFlash database and need to place it in dataset directory?  Could you please guide me how efficiently I can do that.. I was thinking about something like the option you have for eggnog "--download_eggnog" can also be applied for phyloFlash, but this may be for later?

**Comment by rpetit3** [comment]
You're right! That's an oversight on my part, at the moment its assumed you provide the phyloflash database. I agree it should work like GTDB and Eggnog (provide the option to download).

I'll work on adding this. In the meantime your best bet is to create an environment with phyloFlash then run `phyloFlash_makedb.pl --remote` (See: https://github.com/HRGV/phyloFlash#set-up-database-and-run)

**Comment by rpetit3** [comment]
Happy to report in the next version (v1.5.3) you will be able to download phyloflash database (https://github.com/bactopia/bactopia/commit/882cd64ba775e6b794c99df6b5ef0f74692c1c3e)

I added two parameters
```
        --download_phyloflash   Download the latest phyloFlash database, even it exists.

        --yes                   You acknowledge SILVAs license.
```

If you use `--download_phyloflash` it will download the phyloFlash database to the location given by `--phyloflash`. Existing phyloflash databases will be overwritten.

I also added a `--yes` parameter to explicitly say you acknowledge the SILVA license (CC-BY 4.0 https://ftp.arb-silva.de/current/Exports/LICENSE.txt), which is printed to screen. The `phyloFlash_makedb` command has a 'yes/no' prompt and the `--yes` acts as the user typing "yes"
